src/clipboard_handlers/tables_clipboard.ts

```diff
@@ -69,8 +69,16 @@ export class TableClipboardHandler extends AbstractCellClipboardHandler<
           zones.some((z) => isZoneInside(tableZone, z))
         ) {
           copiedTablesIds.add(table.id);
+          let { numberOfRows } = zoneToDimension(tableZone);
+          for (let r = tableZone.top; r <= tableZone.bottom; r++) {
```

Comment:
We can spend 2 more characters for the sake of readability
```suggestion
          for (let row = tableZone.top; row <= tableZone.bottom; row++) {
```

Reply:
This loop is inside another for loop that use the "row" variable ...